### PR TITLE
Sync with staging

### PR DESCRIPTION
**Pull request by sverkoye**
[RDKCMF-8893]  Sync rdkcentral/meta-dac-sdk with stagingrdkm/meta-dac-sdk

### CODE REVIEW

File: conf/layer.conf [hunk 1]
```diff
@@ -16,8 +16,52 @@ DISTRO_FEATURES_remove = " x11"
...
 PREFERRED_VERSION_libepoxy = "1.5.3"
+
+RDK_GIT_PROTOCOL = "https"
+CMF_GIT_PROTOCOL = "https"
+CMF_GIT_BRANCH = "rdk-next"
+CMF_GIT_ROOT = "git://code.rdkcentral.com/r"
+CMF_GITHUB_MASTER_BRANCH ?= "master"
+CMF_GITHUB_ROOT ?= "git://github.com/rdkcentral"
+RDK_RELEASE ?= "1.99"
+LICENSE_FLAGS_WHITELIST += "commercial"
```
Comment (mhughesacn) [inline conf/layer.conf:47]: What is being enabled here?  In rdk.conf (meta-rdk), this is selectively enabled for particular packages.

File: recipes-core/playready-headers/playready-headers_4.2.bb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Playready headers"
+LICENSE = "CLOSED"
```
Comment (mhughesacn) [inline recipes-core/playready-headers/playready-headers_4.2.bb:2]: This is an opensourced repo so it is unusual to refer to a closed license component.  In particular, are some filenames and paths from a non-open zipfile being made visible?

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Hi,
I have a few questions below and also, every new patch file should have a header with at least an indication of the source of the patch (e.g. Source: LGI, or an email etc.)
Thanks, Martin

File: recipes-core/playready-headers/playready-headers_4.2.bb (hunk 2 above)
Comment (sverkoye) [reply recipes-core/playready-headers/playready-headers_4.2.bb]: Removed the playready-headers_4.2 recipe. Updated the readme to point to another location for it.

**Review by mhughesacn — APPROVED** [review]

**Review by bcatrysse — APPROVED** [review]

**Pull request merged**